publisher-sdk/src/androidTest/java/com/criteo/publisher/integration/DfpHeaderBiddingFunctionalTest.java

```diff
@@ -554,39 +563,46 @@ private String loadDfpHtmlInterstitial(InterstitialAdUnit adUnit) throws Excepti
 
     loadBidAndWait(adUnit, builder);
 
-    builder.addTestDevice(AdRequest.DEVICE_ID_EMULATOR);
-    PublisherAdRequest request = builder.build();
+    RequestConfiguration requestConfiguration = new RequestConfiguration.Builder().setTestDeviceIds(Collections.singletonList(AdRequest.DEVICE_ID_EMULATOR)).build();
+    MobileAds.setRequestConfiguration(requestConfiguration);
+
+    AdManagerAdRequest request = builder.build();
 
-    PublisherInterstitialAd publisherInterstitialAd = createDfpInterstitialView();
-    DfpSync dfpSync = new DfpSync(publisherInterstitialAd);
+    DfpSync dfpSync = new DfpSync();
+    SyncAdManagerInterstitialAdLoadCallback syncAdManagerInterstitialAdLoadCallback = dfpSync.getSyncAdManagerInterstitialAdLoadCallback();
```

Comment:
Nit: That's a really long name and the context is quite clear. What do you think about shorten it a little bit ?

Reply:
done